docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/the-git-based-development-process-for-mainframe-development.md

```diff
@@ -34,68 +35,54 @@
 
 Many mainframe development teams follow a release-based or iteration-based process to deliver incremental updates to a pre-defined production runtime.
 
-## Characteristics of mainline-based development with feature branches
+## Characteristics of mainline-based development with feature branches 
+
+As mentioned in the [Introduction](introduction.md) our recommended approach scales very well to support the needs of a range of team sizes, frequency and size of changes, and degrees of concurrent working.
 
-As mentioned in the [Introduction](introduction.md) our recommended approach scales very well to support the needs of a range of
-team sizes, freqency and size of changes, and degrees of concurrent working.
 ### Starting simple
 
 The mainline-based development approach[^3] with *feature* branches is a simple and structured workflow to implement, integrate, and deliver changes with an early integration process flow using a single long-living branch: *main*. Developers work in isolation in *feature* branches to implement changes to the source code, and ideally test the changes in a specific environment.
 
 This approach can be compared to a trunk-based branching model that leverages feature branches. A similar workflow like outlined in this publication is also documented by Microsoft without giving it a name[^5].
 
 <!-- this paragraph would make an ideal 'annotation', callout or sidebar if writing for an SSG like Mkdocs. -->
-(You may also see reference to *topic* branches, for example [here](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows).
-Feature branches can be considered as one sub-type of *topic* branches with branches for *bug-fixes* being another sub-type.
-Both feature branches and bug-fix branches are expected to be relatively short-lived and ultimately to merge into a long-lived branch.)
+(You may also see reference to *topic* branches, for example [here](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows). Feature branches can be considered as one sub-type of *topic* branches with branches for *bug-fixes* being another sub-type. Both feature branches and bug-fix branches are expected to be relatively short-lived and ultimately to merge into a long-lived branch.)
```

Comment:
@M-DLB the above section still contains footnotes, which no longer exist. Is there propably a commit missing?

Reply:
That part was also in the original document, so I didn't touch it.
But yes I agree it should not at least be between parenthesis. There is one comment (from @ijmitch I guess) in the original markdown file that says: "this paragraph would make an ideal 'annotation', callout or sidebar if writing for an SSG like Mkdocs.".
Maybe something @lauren-li can implement when migrating to the final version?

Reply:
Following the result of our discussion the related [Slack thread](https://ibm-systems-z.slack.com/archives/C04JPQKF3UP/p1697016613178259), we will use Docusaurus's "Note" [admonition](https://docusaurus.io/docs/markdown-features/admonitions) in this case.